class.jetpack-debugger.php

```diff
@@ -18,6 +18,27 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	static function seconds_to_time( $seconds ) {
```

Comment:
true, but it doesn't give you info if the sync has been delayed for days or weeks even. Hopefully that shouldn't happen.